chrome/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "author": "Libor Polčák, Martin Bednář, Martin Timko, Pavel Pohner, Peter Horňák",
```

Comment:
Do not forget to put your name here and to any file that you change, see also credits.md.

This is very important for the final submission of the thesis.